src/gui/sharedialog.h

```diff
@@ -52,7 +52,7 @@ class ShareDialog : public QDialog
     QSize minimumSizeHint() const override;
 
 private slots:
-    void slotPropfindReceived(const QMap<QString, QString> &result);
+    void slotPropfindReceived(const QString &, const QMap<QString, QString> &result);
```

Comment:
Wouldn't it be better style to adapt the connections?

Alternatively we might add a second signature with a FIXME that calls the original one.

Reply:
An additional signal would need to be emitted.
How would you change the connect?

Reply:
There's the option of connecting a lambda which calls the slot correctly. But that's less efficient.